libraries/SuplaDevice/supla/network/esp32_wifi.h

```diff
@@ -73,6 +73,8 @@ class ESP32Wifi : public Supla::Network {
   }
 
   bool connect(const char *server, int port) {
```

Comment:
add default -1 for port
Then do following check in this method:
if (port == -1) { port = 2015 }